package customExceptionandtry_catch_finally;public class CustomException {
}
